src/app/pages/partners/results/results.tsx

```diff
@@ -202,13 +200,51 @@ function resultEntry(pd: PartnerData) {
         cost: pd.affordability_cost,
         rating: pd.average_rating.rating__avg,
         ratingCount: pd.rating_count,
-        partnershipLevel: pd.partnership_level,
+        partnershipLevel: pd.partnership_level as string,
```

Comment:
I think it's already typed as string here https://github.com/openstax/os-webview/blob/add-startups-sidebar-to-partners/src/app/pages/partners/results/results.tsx#L52

Reply:
I made startups display as a normal grid-group if it's there is only one other grid-group.
I think I've got the spacing all fixed up.
The "as string" was leftover from when I had it defined as string | null. Good catch, thanks.
I removed some padding on the mobile-controls row, which was enough to make it good down to 320px screen width.

Reply:
Yeah this is a lot better, nice idea to switch up the display based on the available groups.